### ISSUE TITLE
output files

### ISSUE THREAD

**Issue by aremazeilles** [CentroEPiaggio/dysturbance#10]
When running the command under Matlab:

```matlab
run_local_pi("./tests/data/input/subject_1/protocol_1/subject_1_cond_1008203500/raw_data_input/subject_1_cond_1008203500_run_0_platformData.csv", "./tests/data/input/subject_1/protocol_1/subject_1_cond_1008203500/raw_data_input/subject_1_cond_1008203500_testbed.yaml", "./tests/data/input/subject_1/subject_1_info.yaml","./out")
```
I get the following result in folder `out`:

```term
tree out          
out
└── subject_1
    └── protocol_1
        └── subject_1_cond_1008203500
            ├── Local_PI
            │   ├── Absorbed_energy_subject_1_cond_1008203500_run_0_pp_platformData.yaml
            │   ├── Fall_check_subject_1_cond_1008203500_run_0_platformData.yaml
            │   └── Stability_margin_subject_1_cond_1008203500_run_0_pp_platformData.yaml
            └── Preprocessed_data
                ├── protocol_check.csv
                └── subject_1_cond_1008203500_run_0_pp_platformData.csv

5 directories, 5 files

```

Looking only at this, I assume you algorithm generated 3 PIs, `absorbed energy`, `fall_check`, and `stability_margin`, is that correct?

If so, we would only need to get in the output older: `absorbed_energy.yaml`, `fall_check.yaml`, and `stability_margin.yaml`. 
Is it possible to adjust the name of the output files this way?

Then we should discuss on the raw / preprocessed file that you use as input / generates as output.

The idea is that our PI manager will iteratively call your program for each subject / condition / run detected. He will get all PI files, and he will rename the file according to the subject / cond / run. We think it is better this way, because this way the algorithm does not make any assumption on the filenaming pattern.

Do you see my point ?

**Comment by aremazeilles** [comment]
ping @simomonte @qbrobotics-AT

**Comment by aremazeilles** [comment]
ping

That would be great to consider this, to progress on the integration

**Comment by aremazeilles** [comment]
ping

**Comment by simomonte** [comment]
Hi Anthony! Sorry for the late reply!
I see your point and I can adjust the code to match your expected output. I will work on this as soon as I can.
Best Regards.

**Comment by aremazeilles** [comment]
Hello, just checking whether anythng could advance there. Thanks

**Comment by simomonte** [comment]
Hi Anthony! I am planning to solve this issue this week. Sorry for the delay.
Best, Simone

**Comment by aremazeilles** [comment]
Hello, any chance of update?

**Comment by simomonte** [comment]
Hi Anthony! I am deeply sorry for the delay! I just commit the change that you suggested! I hope that it matches your expectations!

**Comment by aremazeilles** [comment]
Hello @simomonte I somehow missed your swift reply back in November.
It seems you did not upload the change you mention. The [last commit](https://github.com/CentroEPiaggio/dysturbance/commits/master) on the code is of the 27 of April. Maybe you just forgot to push?

**Comment by simomonte** [comment]
Hi Anthony! No, I am sorry! But it appears to me that a conflictual branch has been created while I uploaded the changes in November. As soon as I understood the reasons for this, I will correct them and ask for a merge.

**Comment by aremazeilles** [comment]
Happy new year! Any news? Do you need support for moving this?

**Comment by simomonte** [comment]
Hi Anthony! Sorry for the delay in the reply! I saw that during the submit of the correction that you requested, github created a new branch (conflictual Branch). I requested for a merge of it on 20th February! Did it solve the issue?

**Comment by aremazeilles** [comment]
Hello 
I did not see the merge request. I see indeed the branch named conflictual branch, but no merge request associated.
I will try this other branch and see whether it is better with it, ok?

**Comment by simomonte** [comment]
Ok, let me know if the branch is good for the modification asked. If it is ok, I will try to request the merge again!

**Comment by aremazeilles** [comment]
Hi
I made some trials yesterday. Under Octave, I get some issues with the use of `readtable` which is not implemented in Octave (at least in the version I use). 
I tried through Matlab, and I think I managed to generate similar results as you have stored in the reference output.
But the output folder hierearchy remains in the spirit of what I commented at the beginning of this issue. If that is ok with you, I will check whether I can directly adjust your code to get it compatible with our expectation.

Then I saw that this `conflicted_branch` is _2 commits ahead, 2 commits behind master_ . Meaning the master has changes not present in that branch. I hope this will not be an issue for merging later on
